Unix/etc/omiserver.conf

```diff
@@ -58,3 +58,5 @@ NoSSLCompression=false
 #NtlmCredsFile=PATH
 # For example
 #NtlmCredsFile=/etc/opt/omi/.creds/ntlm
+nonroot=false
+service=test1
```

Comment:
I guess this is a "fake" omiserver.conf file (not the one we ship to customers). But still, do we really want lines 61/62 here? Shouldn't unit tests set this up if that's what they want?